Admin/View/AutomationViewBuilder.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AutomationBundle\Admin\View;
+
+use Sulu\Bundle\AdminBundle\Admin\View\FormOverlayListViewBuilder;
+use Sulu\Bundle\AdminBundle\Admin\View\ToolbarAction;
+use Sulu\Bundle\AutomationBundle\Entity\Task;
+use Sulu\Bundle\PageBundle\Admin\PageAdmin;
+
+class AutomationViewBuilder extends FormOverlayListViewBuilder implements AutomationViewBuilderInterface
+{
+    public function __construct(string $name, string $path)
+    {
+        parent::__construct($name, $path);
+
+        $listToolbarActions = [
+            new ToolbarAction('sulu_admin.add'),
+            new ToolbarAction('sulu_admin.delete'),
+        ];
+
+        $this->setResourceKey(Task::RESOURCE_KEY)
+            ->setListKey(Task::RESOURCE_KEY)
+            ->setFormKey('task_details')
+            ->setTabTitle('sulu_automation.automation')
+            ->addToolbarActions($listToolbarActions)
+            ->addListAdapters(['table'])
+            ->setTabOrder(4096)
+            ->addRouterAttributesToFormRequest(['id' => 'entity-id'])
+            ->addRouterAttributesToListRequest(['id' => 'entity-id'])
+            ->setParent(PageAdmin::EDIT_FORM_VIEW);
```

Comment:
i guess the intention of creating a `AutomationViewBuilder` class is that you can easily add an automation tab for your own entity? in that case we should not hardcode the parent route